commit 92af07adfb1e06fe1b055fbcf5ba61159637cd73
Author: Michael Tremer <michael.tremer@ipfire.org>
Date:   Wed Jun 3 16:33:44 2020 +0000

    location-exporter: Warn, but do not fail on invalid input
    
    Signed-off-by: Michael Tremer <michael.tremer@ipfire.org>

diff --git a/src/python/location-exporter.in b/src/python/location-exporter.in
index 5454561..d82f1d3 100644
--- a/src/python/location-exporter.in
+++ b/src/python/location-exporter.in
@@ -270,8 +270,12 @@ class CLI(object):
 				countries.append(object)
 
 			else:
-				log.error("Invalid argument: %s" % object)
-				return 2
+				log.warning("Invalid argument: %s" % object)
+				continue
+
+		if not countries and not asns:
+			log.error("Nothing to export")
+			return 2
 
 		# Open the database
 		try:
